### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lzehl**
modified similar to the one from core

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,17 @@
 </a>
 
 # openMINDS_SANDS
+The openMINDS_SANDS repository repository hosts one of the metadata models of the **openMINDS metadata framework**. It defines modular metadata schemas for **s**patial **a**nchoring of **n**euroscience **d**ata **s**tructures.
```
Comment (UlrikeS91) [inline README.md]: "The openMINDS_SANDS repository repository [...]" remove one "repository"

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]
One small typo. Otherwise fine to merge.

**Comment by lzehl** [comment]
@UlrikeS91 this PR was outdated in any case. I've made now the updates.

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]
The svgs don't seem to match the pngs, could you check if you uploaded the correct svgs or pngs depending on which one is the one you want to keep?

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**